src/Providers/RoutingServiceProvider.php

```diff
@@ -7,22 +7,12 @@
 
 class RoutingServiceProvider extends ServiceProvider
 {
-    public function register()
-    {
-        //
-    }
-
-    public function boot()
-    {
-        $this->routesInDirectory();
-    }
-
     /**
-     * Organise Routes.
+     * Boot the service provider.
      *
-     * @param string $app
+     * @return void
      */
-    private function routesInDirectory($app = '')
+    public function boot()
     {
         $routeDir = app_path('routes/'.$app.($app !== '' ? '/' : null));
```

Comment:
Nop I don't know why it has changes. https://github.com/cachethq/Cachet/blob/master/src/Providers/RoutingServiceProvider.php